### PR TITLE
Added Supermicro vendor theme

### PR DESCRIPTION
**Pull request by Jackson5551**
# Description

Added Supermicro vendor theme for use on Supermicro server and workstation boards.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (correct file and folder structure)
- [x] The background image for theme follows naming convention: `theme_name.png`
- [x] I updated [`themes.json`](https://github.com/AdisonCavani/distro-grub-themes/blob/master/themes.json) file (JSON database)

### CODE REVIEW

**Comment by Jackson5551** [comment]
I have the .psd file as well and will provide that upon request.

File: themes.json
Comment (AdisonCavani) [inline themes.json:1]: Move this to the end of the file:
```json
{
      "name": "Supermicro",
      "versionAdded": "v3.3"
}
```

**Review by AdisonCavani — CHANGES_REQUESTED** [review]

File: themes.json
Comment (Jackson5551) [reply themes.json:1]: Done

**Review by AdisonCavani — APPROVED** [review]

**Comment by AdisonCavani** [comment]
You can provide the `psd` file here

**Comment by Jackson5551** [comment]
[supermicro_1080p.zip](https://github.com/AdisonCavani/distro-grub-themes/files/12906929/supermicro_1080p.zip)

**Pull request merged**